src/docs/sdk/cache.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Cache
+sidebar_order: 20
+---
+
+The SDK's caching integration is agnostic to the underlying engine used. In most cases, the SDK will instrument existing cache abstractions in certain frameworks or libraries.
+
+## Span conventions
+
+### Span Operations
+
+| Span OP | Description |
+|:--|:--|
+| `cache.get` | A single item or multiple items are retrieved from the cache |
+| `cache.put` | A single item or multiple items are written to the cache |
+| `cache.remove` | A single item or multiple items are removed from the cache |
+| `cache.flush` | The entire content of the cache is deleted |
```

Comment:
No strong opinion here, just made the OPs used more concise.

Reply:
`cache.get_item` is used in Python for around 11 months right now, so we need to also support this on the Sentry backend to be backwards compatible.

Reply:
To add to @antonpirker point, we document the existing django cache operations here, and it's what ingestion expects rn.